sds/src/secondary_validation/lithuanian_personal_identification_number_checksum.rs

```diff
@@ -0,0 +1,72 @@
+use crate::secondary_validation::Validator;
+
+const ROUND_1_WEIGHTS: &[u32] = &[1, 2, 3, 4, 5, 6, 7, 8, 9, 1];
+const ROUND_2_WEIGHTS: &[u32] = &[3, 4, 5, 6, 7, 8, 9, 1, 2, 3];
+
+pub struct LithuanianPersonalIdentificationNumberChecksum;
+
+impl Validator for LithuanianPersonalIdentificationNumberChecksum {
+    fn is_valid_match(&self, regex_match: &str) -> bool {
+        if regex_match.chars().any(|c| !c.is_ascii_digit()) {
+            return false;
+        }
```

Comment:
We could ignore separators to make this a bit more future proof in case the regex is updated in the future to contain some of them.

As all characters are digits, they could be converted right away with 
```rust
let mut digits = regex_match.chars().filter_map(|c| c.to_digit(10));
```